src/Contract/Transaction.purs

```diff
@@ -171,4 +183,11 @@ balanceTx = wrap <<< BalanceTx.balanceTx
 
 -- | Attempts to balance an `UnbalancedTx` hushing the error.
 balanceTxM :: UnbalancedTx -> Contract (Maybe Transaction)
-balanceTxM = map hush <<< balanceTx
\ No newline at end of file
+balanceTxM = map hush <<< balanceTx
+
+finalizeTx
```

Comment:
a docstring? also I think that we should put some information on why the finalization takes place on the server, what is it or why do we need it